src/client/pages/Offer/Introduction/Sidebar/CancellationOptions.tsx

```diff
@@ -105,15 +105,13 @@ type CancellationOptionsProps = {
 }
 
 export const CancellationOptions = ({
-  loadingQuoteIds,
+  isDisabled,
   quotes,
   onToggleCancellationOption,
 }: CancellationOptionsProps) => {
-  const isDisabled = loadingQuoteIds.length > 0
-
   return (
     <>
-      {quotes.map((quote) => {
+      {quotes?.map((quote) => {
```

Comment:
Why does this change? Seems like the prop types don't allow passing `quotes` as undefined, no?

Reply:
that's something I forgot to undo while developing, sorry